packages/Tabs/src/Tabs.js

```diff
@@ -61,43 +72,64 @@ const Tabs = props => {
     activeIndex,
     kind,
     currentFocusIndex,
+    hasInsetFocusStyle,
+    hasTruncation,
+    tabHeight,
+    isVertical,
     onClickTab,
     onKeyDown,
     isDisabled,
     setTabListRef,
+    size,
   };
 
-  return <TabsContext.Provider value={contextValue}>{props.children}</TabsContext.Provider>;
-};
+  return <TabsContext.Provider value={contextValue}>{children}</TabsContext.Provider>;
+}
 
-Tabs.types = {
-  kind: constants.kind,
-};
+Tabs.types = types;
 
-const propTypes = {
-  /** Determine the styling of the tab */
+Tabs.propTypes = {
+  /** The visual theme of the tabs list. */
   kind: PropTypes.oneOf([Tabs.types.kind.PRIMARY, Tabs.types.kind.SECONDARY]),
-  /** Children of the Tab */
+
+  /** Expects Tabs.List and Tabs.Panels. */
   children: PropTypes.node.isRequired,
-  /** Sets what tab index is active by default */
+
+  /** Sets what tabindex is active by default. */
   defaultIndex: PropTypes.number,
-  /** If the tab is disabled */
+
+  /** If the visual focus ring for the tabs should be displayed with an inset style. */
+  hasInsetFocusStyle: PropTypes.bool,
+
+  /** Tab labels will be truncated when they run out of space instead of breaking to multiple lines (ignored when isVertical is false). */
+  hasTruncation: PropTypes.bool,
+
+  /** If the tabs are all disabled. */
   isDisabled: PropTypes.bool,
+
+  /** If the tabs are stacked vertically. */
+  isVertical: PropTypes.bool,
+
+  /** Size of the tab label text.  */
+  size: PropTypes.oneOf([Tabs.types.size.MEDIUM, Tabs.types.size.LARGE]),
+
+  /** Height, in pixels, of the tabs (ignored when isVertical is true). */
+  tabHeight: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
```

Comment:
Oh, this is another change, the `tabHeight` can be a number or a string (CSS value).  But I need to update the comment description.